### ISSUE TITLE
Support for Blender files

### ISSUE THREAD

**Issue by JakubKahovec** [dmsurti/AssimpKit#70]
Hi Deepak,
do you plan to add support for Blender files (.blend), which are supported by the Assimp library
Thanks

**Comment by dmsurti** [comment]
@JakubKahovec Yes, as of version 1.2.0 29 files are supported. I now plan to add the pending files. Blender is the next one, as I am also a Blender user :-). Thanks for +1 to blend file support.

FYI, most likely next month will see Blender supported added.

**Comment by JakubKahovec** [comment]
Hi Deepak,
how does it go with the support for Blender ?
Thanks

**Comment by dmsurti** [comment]
I have done some testing with with some complex Blender files and the results are only satisfactory (both geometry and textures/materials). I might just release it as an alpha support so it can be tested out in the wild.

It will be great if you can share some blender files for testing.
